oars/models/enrollment/enrollment_events.sql

```diff
@@ -0,0 +1,17 @@
+select
+  event_id,
+  emission_time,
+  actor_id,
+  object_id,
+  course_id,
+  org,
+  verb_id,
+  event_type,
+  JSON_VALUE(event_str, '$.object.definition.extensions."https://w3id.org/xapi/acrossx/extensions/type"') as enrollment_mode
```

Comment:
Are you actually getting any values for this? It's all coming up empty for me.

Reply:
Yup, it looks like it's coming through for my manual enrollments. I can see four enrollments for the demo course with an `enrollment_mode` of `audit` in the last two weeks.

Reply:
I bet mine are weird because I'm the course author / course staff, something to keep in mind.